packages/server/src/user/user.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Mutation, Args, Resolver, Query } from '@nestjs/graphql';
```

Comment:
This resolver is not registered with Nest.js, add UserResolver to the UserModule as a provider